metricbeat/module/kubernetes/_meta/kibana/7/visualization/da1ff7c0-30ed-11e7-b9e5-2b5b07213ab3-ecs.json

```diff
@@ -40,47 +47,68 @@
                 "id": "2fe9d3b0-30d5-11e7-8df8-6d3604a72912",
                 "index_pattern": "metricbeat-*",
                 "interval": "auto",
+                "isModelInvalid": false,
```

Comment:
is this field part of `Panel Option`? could you add a screenshot for the page where this is defined?

Reply:
I cannot find it in the view tbh, @flash1293 do you have an answer handy for this?